client/src/includes/a11y-result.ts

```diff
@@ -203,7 +204,7 @@ export const renderA11yResults = (
         // Special-case when displaying accessibility results within the admin interface.
         const isInCMS = node.target[0] === '#w-preview-iframe';
         const selectorName = toSelector(
-          isInCMS ? node.target[1] : node.target[0],
+          node.target.filter((target) => target !== '#w-preview-iframe'),
```

Comment:
This ensures the selector is displayed correctly when you use multi-layered iframe approach for headless